### PR TITLE
Pl-129904 cgroup pressure information

### PR DESCRIPTION
**Pull request by elikoga**
@flyingcircusio/release-managers

## Release process

Impact:

Changelog:

## Security implications
None

- [x] [Security requirements](https://wiki.flyingcircus.io/System_Development_Guideline#Security_requirement_principles_and_testing) defined? (WHERE)
 - Exposes performance, not security relevant metrics to grafana dashboard
- [x] Security requirements tested? (EVIDENCE)
 - New metrics show up in grafana
 - Test to confirm metrics don't show up unless opted in

### CODE REVIEW

**Comment by elikoga** [comment]
Added option for regex

File: nixos/services/telegraf/psi_input.nix [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+let 
+  inherit (lib) types;
+  psi_cgroup_regex = lib.concatStringsSep "|" config.flyingcircus.services.telegraf.psi_cgroup_regex;
+in {
+  options.flyingcircus.services.telegraf.psi_cgroup_regex = lib.mkOption {
+    type = types.listOf types.str;
+    default = [ ];
+  };
+  config = {
+    flyingcircus.services.telegraf.psi_cgroup_regex = [ "^/system.slice$" "^/system.slice.*service$" "^/user.slice$" ];
```
Comment (dpausp) [inline nixos/services/telegraf/psi_input.nix]: camelCase for option names

**Review by dpausp — CHANGES_REQUESTED** [review]
* the package name has a typo: `fc-telegraf-collet-psi` (from the previous PR)
*  commits should be squashed, have a longer commit message and contain the ticket ID from youtrack

File: nixos/services/telegraf/psi_input.nix [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
+let 
+  inherit (lib) types;
+  psiCgroupRegex = lib.concatStringsSep "|" config.flyingcircus.services.telegraf.psiCgroupRegex;
+in {
+  options.flyingcircus.services.telegraf.psiCgroupRegex = lib.mkOption {
+    type = types.listOf types.str;
+    default = [ ];
+  };
+  config = {
+    flyingcircus.services.telegraf.psiCgroupRegex = [ "^/system.slice$" "^/system.slice.*service$" "^/user.slice$" ];
```
Comment (dpausp) [inline nixos/services/telegraf/psi_input.nix]: regex list should be empty by default, would produce too many metrics which our central statshost couldn't handle. Use the default regex as option example instead (and an option description would also be nice).

**Review by dpausp — CHANGES_REQUESTED** [review]

**Pull request merged**